### DISCUSSION TITLE
Ability to open URIs (local files)

### DISCUSSION THREAD

**Discussion by onstrt** [tsujan/FeatherPad#747] [Ideas]
The "Ability to open URLs with appropriate app" is a great feature. Something similar for local files would be a good enhancement.

**Comment by tsujan** [comment]
Already supported with `file:///`.

"Appropriate apps" are defined by the DE, not by FeatherPad; FeatherPad isn't a file manager.

**Comment by onstrt** [comment]
Thanks,  I'd forgotten...

I think that because I didn't see it on the list of attributes I presumed it did not support it.

I've used it before, time ago ,but I was looking at a file that was missing the  'file:///' before the uri.

 

Having just migrated to ubuntu 22.04 a Featherpad file is a good canditate for a "non-firefox" bookmark file.

(Previously I used argos but that is not supported on 2204.)


 

Thanks again for a lovely editor.

 

Sent: Tuesday, June 20, 2023 at 2:52 PM

**Reply by tsujan** [reply]
> ... is a good canditate for a "non-firefox" bookmark file

I sometimes use it as an audio/video playlist.

However, since a URL cannot contain spaces, they should be replaced by "%20".

Also, some apps may not accept commas, in which case, they can be replaced by "%2c"
